Apps/WebClient/src/ClientApp/src/views/loginCallback.vue

```diff
@@ -37,6 +43,12 @@ export default class LoginCallbackView extends Vue {
                     this.checkRegistration({ hdid: this.user.hdid }).then(
                         () => {
                             this.$router.push({ path: redirectPath });
+                            if (
+                                !this.isSidebarOpen &&
```

Comment:
Instead how about having the sidebar expanded by default?

Reply:
If that were the case, every time a user refreshed the sidebar would expand. If this is expected behaviour then I can change it but this implementation follows the story's requirements exactly

Reply:
We would also need to implement logic to only expand on mobile anyways

Reply:
Well it would expand it on refresh too. Making the default extended.

Reply:
I was thinking about it and it would be the same for mobile. Something like, if is mobile set the default state to collapse, otherwise is extended. I don't think having the animation being shown every time would be necessary

Reply:
Chatted win nino and we agree that it should be expanded by default